r/R/dplyr-arrange.R

```diff
@@ -20,7 +20,10 @@
 
 arrange.arrow_dplyr_query <- function(.data, ..., .by_group = FALSE) {
   call <- match.call()
-  exprs <- quos(...)
+
+  expression_list <- expand_across(.data, quos(...))
+  exprs <- ensure_named_exprs(expression_list)
```

Comment:
Can this line be removed? I don't think arrange expressions have to have names.

Reply:
Thank you for taking a look. I have removed the unnecessary lines.